analytics_data_api/v0/tests/test_connections.py

```diff
@@ -0,0 +1,35 @@
+from django.test import TestCase
+from django.test.utils import override_settings
+from elasticsearch.exceptions import ElasticsearchException
+from mock import patch
+
+from analytics_data_api.v0.connections import BotoHttpConnection, ESConnection
+
+
+class ESConnectionTests(TestCase):
```

Comment:
I wonder if we shouldn't test the timeout case as well? Not sure what boto does when there is a timeout... but we should emulate it and make sure this connection code handles the case properly. Or perhaps all we need is a generic test that makes sure this "does the right thing" when an unexpected exception occurs in boto's request logic.